crates/oxc_query/src/schema.graphql

```diff
@@ -323,13 +324,13 @@ A single VariableDeclarator. (const/let/var)!
 If you have multiple variables assigned at once with commas,
 this will fire once per assignment.
 """
-type VariableDeclarationAST implements VariableDeclaration {
+type VariableDeclarationAST implements VariableDeclaration & ASTNode & HasSpan {
   left: AssignmentType!
   # ASTNode
   parent: ASTNode
   ancestor: [ASTNode!]!
   # HasSpan
-  entire_span: Span!
```

Comment:
The structure here makes me think this type was intended to implement `ASTNode` and `HasSpan` previously, but did not.

Could you proactively look for other places in the schema where such a bug might have happened?